roles/longhorn/tasks/deploy.yml

```diff
@@ -34,3 +34,17 @@
             AWS_SECRET_ACCESS_KEY: "{{ longhorn_s3_backup_key_secret | b64encode }}"
             AWS_ENDPOINTS: "{{ longhorn_s3_backup_endpoint | b64encode }}"
         state: "{{ 'present' if longhorn_s3_backup_enabled else 'absent' }}"
+
+    - name: Unset default storage class for local-path # noqa var-spacing
+      ansible.builtin.command: >
```

Comment:
[`kubernetes.core.k8s_json_patch`](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_json_patch_module.html) should be used here.

Reply:
Didn't realise that was a module - I'll now update it using that.

Reply:
Are the `noqa` statements still needed with this change?

Reply:
Doesn't look like it - changed it back now